packages/agent/src/agent/http/index.ts

```diff
@@ -224,7 +224,14 @@ export class HttpAgent implements Agent {
         );
       }
       // Mainnet and local will have the api route available
-      const knownHosts = ['ic0.app', 'icp0.io', '127.0.0.1', 'localhost'];
+      const knownHosts = [
+        'ic0.app',
+        'icp0.io',
+        '127.0.0.1',
+        'localhost',
+        'github.dev',
```

Comment:
Out of curiosity, has it been reviewed by prodsec / does it require a security review?

Reply:
To my understanding, this wouldn't have security implications. This change only affects remote dev environments which originally had the same behavior prior to 0.19.1.

Reply:
Thanks for the feedback.